go/ql/test/query-tests/Security/CWE-918/websocket.go

```diff
@@ -96,7 +96,7 @@ func test() {
 	http.HandleFunc("/ex5", func(w http.ResponseWriter, r *http.Request) {
 		untrustedInput := r.Referer()
 
-		// good as input is tested againt regex
+		// good as input is tested against regex
```

Comment:
Similarly, but a different spelling. And I don't think I'm actively ignoring `-tests`...